conf/config.yaml

```diff
@@ -9,48 +10,26 @@ hydra:
   output_subdir: ${hydra.run.dir}/hydra
 
 pipeline:
-    - wir_table_generator
-    - wir_blob_data_generator
-    - process_blob_analysis
-    - process_tables_analysis
-    - append_datetime
+    # - wir_table_generator
+    # - wir_blob_data_generator
+    # - process_blob_analysis
+    # - process_tables_analysis
+    # - append_datetime
     # - create_batches
     - report
-    - image_inspection
-    - plot_by_season
-
+    # - image_inspection
+    # - plot_by_season
 
 general:
   task: # Leave empty as placeholder
   workdir: ${hydra:runtime.cwd}  # where the code is executed
 
 pipeline_keys: ${general.workdir}/keys/authorized_keys.yaml
 
-data:
-  datadir: ${general.workdir}/data
-  tablesdir: ${data.datadir}/tables/${job.job_now_date}
-  blobsdir: ${data.datadir}/blobs/${job.job_now_date}
-  processed_datadir: ${data.datadir}/processed_tables/${job.job_now_date}
-  persistent_datadir: ${data.datadir}/persistent_tables
-  permanent_merged_table: ${data.persistent_datadir}/merged_blobs_tables_metadata_permanent.csv
-  longterm_storage: /mnt/research-projects/r/raatwell/longterm_images3/field-batches
-
-
-temp:
-  temp_json_data: ${general.workdir}/tempdata/json_data
-  temp_jpegs_data: ${general.workdir}/tempdata/jpeg_data
-  final_json_dir: ${general.workdir}/tempdata/final_jsons
-  temp_image_dir: ${general.workdir}/tempdata/temp_images
-
-report:
-  reportdir: ${general.workdir}/report
-  report_plots: ${report.reportdir}/${job.job_now_date}/plots
-  plots_all_years: ${report.reportdir}/${job.job_now_date}/plots/plots_all_years
-  plots_current_season: ${report.reportdir}/${job.job_now_date}/plots/plots_current_season
-  inspectdir: ${report.reportdir}/${job.job_now_date}/inspection
-  missing_batch_folders: ${report.reportdir}/${job.job_now_date}/missing_raws.csv
-  uploads_7days: ${report.reportdir}/${job.job_now_date}/num_uploads_last_7days_by_state.csv
-  preprocessing_analysis: ${report.reportdir}/${job.job_now_date}/preprocessing_analysis.csv
+inspection:
+  num_past_days_to_inspect: 180
+  num_past_days_for_report: 10
+  num_images_to_inspect: 10
```

Comment:
this should at least be 50 by default if we have a dedicated person reviewing images.